app/page.tsx

```diff
@@ -2,100 +2,56 @@ import Image from "next/image";
+          </h1>
+          <p className="text-xl text-muted-foreground">
+            Safe, reliable transportation for your four-legged family members.
+            From vet visits to grooming appointments, we've got your pup
+            covered.
+          </p>
+        </div>
+
+        <div className="grid grid-cols-1 sm:grid-cols-2 gap-8 mb-16">
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">For Pet Parents</h2>
+            <ul className="space-y-3">
+              <li>✓ Real-time tracking of your dog's journey</li>
+              <li>✓ Vetted and trained pet transport specialists</li>
+              <li>✓ Flexible scheduling for routine appointments</li>
+              <li>✓ Safe, comfortable, climate-controlled vehicles</li>
+            </ul>
+          </div>
+
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">How It Works</h2>
+            <ul className="space-y-3">
+              <li>1. Book a ride through our easy-to-use app</li>
+              <li>2. Meet your certified pet transport specialist</li>
+              <li>3. Track your dog's journey in real-time</li>
+              <li>4. Receive updates throughout the trip</li>
+            </ul>
+          </div>
+        </div>
 
-        <div className="flex gap-4 items-center flex-col sm:flex-row">
+        <div className="flex justify-center gap-4">
           <a
-            className="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5"
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
+            href="#"
+            className="rounded-full bg-foreground text-background px-8 py-3 font-semibold hover:bg-foreground/90"
           >
-            <Image
-              className="dark:invert"
-              src="/vercel.svg"
-              alt="Vercel logomark"
-              width={20}
-              height={20}
-            />
-            Deploy now
+            Book a Ride
           </a>
           <a
-            className="rounded-full border border-solid border-black/[.08] dark:border-white/[.145] transition-colors flex items-center justify-center hover:bg-[#f2f2f2] dark:hover:bg-[#1a1a1a] hover:border-transparent text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:min-w-44"
-            href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
+            href="#"
+            className="rounded-full border px-8 py-3 font-semibold hover:bg-muted"
           >
-            Read our docs
+            Learn More
```

Comment:
Hun, where's the love for the user feedback 😲? Let's add hover, focus and active states on these links to make them extra interactive and slayin' 🥳💥